test/testdefs.jl

```diff
@@ -1,7 +1,7 @@
 # This file is a part of Julia. License is MIT: http://julialang.org/license
 
 function runtests(name, isolate=true)
-    Base.Test.DISABLE_TESTSET_PRINTING[] = true
+    Base.Test.testset_print_enable(false)
```

Comment:
https://github.com/JuliaLang/julia/pull/20221/files?w=1,This is not intended to be a public api where you can disable and enable at will. Right now it is like 0.5 with an escape hatch for base to disable it.